x/cosmwasm/bindings/msg.go

```diff
@@ -0,0 +1,23 @@
+package cosmwasm
+
+type OsmosisMsg struct {
+	/// Contracts can mint native tokens that have an auto-generated denom
+	/// namespaced under the contract's address. A contract may create any number
+	/// of independent sub-denoms.
+	MintTokens *MintTokens `json:"mint_tokens,omitempty"`
+	/// Swap over one or more pools
+	Swap *SwapMsg `json:"swap,omitempty"`
+}
+
+type MintTokens struct {
+	/// Must be 2-32 alphanumeric characters
+	SubDenom  string `json:"sub_denom"`
```

Comment:
Okay, depends on https://github.com/confio/osmosis-bindings/pull/8. Fair enough, just add a comment somewhere in the PR, that we need to merge and tag that.